release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -72,6 +72,27 @@ For more information, see xref:../installing/installing_aws/installing-aws-vpc.a
 [id="ocp-4-14-web-console"]
 === Web console
 
+[id="ocp-4-14-administrator-perspective"]
+==== Administrator Perspective
+
+With this release, there are several updates to the *Administrator* perspective of the web console. You can:
```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) we should avoid introducing lists with incomplete sentences, can you reword to something like this?

```suggestion
With this release, there are several updates to the *Administrator* perspective of the web console. You can now perform the following actions:
```